Orso.Arpa.Domain/Logic/Me/Details.cs

```diff
@@ -10,6 +11,16 @@ public static class Details
     {
         public class Query : IRequest<User> { }
 
+        public class QueryById : IRequest<User>
```

Comment:
Für diesen Query fehlt der entsprechende Handler. Bitte pack doch den Query inklusive Handler in ein separates File.